frontend/app/overview/vocabulary/defined-term.component.js

```diff
@@ -10,7 +10,26 @@ export default function DefinedTerm(props) {
   const containerRef = useRef(null);
   const alwaysLink =
     useContext(DefinedTermContext).alwaysLink || props.alwaysLink;
+
+  const openTooltip = () => {
```

Comment:
Why did these functions get moved, and why switched to consts + arrow function?

Reply:
No reason. I think I was just fooling around a bit. I’ll change the toggles back. There’s an eslint warning for the event function and that’s why I moved it in the useEffect.

Reply:
I needed the `openTooltip` in the useEffect so that's why I moved them up. Would you like me change them back to declarations?

Reply:
if they are declarations, they are hoisted and can be used anywhere. This is actually why I usually use `function() {}` instead of `const thing = () => {}`.

I don't really mind if you write your own code to use `const thing = () => {}`, but yeah let's not change all of the existing components to it.

Reply:
Ok! Sounds good.